Gopkg.toml

```diff
@@ -7,18 +7,18 @@ required = [
 [[constraint]]
   name = "github.com/tektoncd/pipeline"
   branch = "master"
-  
+
 [[override]]
   name = "k8s.io/client-go"
-  version = "kubernetes-1.12.6"
+  version = "kubernetes-1.13.4"
 
 [[override]]
   name = "k8s.io/apimachinery"
-  version = "kubernetes-1.12.6"
+  version = "kubernetes-1.13.4"
 
 [[override]]
   name = "k8s.io/api"
-  version = "kubernetes-1.12.6"
+  version = "kubernetes-1.13.4"
```

Comment:
Having a look again as we think we've done everything for Tekton 0.8, Is this gonna work if we want to still support OCP 4.2 which uses Kube 1.12?

We want to do a 0.5.0 release for Pipelines 0.10 and Triggers 0.2 but for OpenShift 4.2

Reply:
I will work :stuck_out_tongue: 0.9.0 is already depending on 1.13. Depending on this doesn't mean we can't target lower kubernetes version :angel:

Reply:
Excellent, let's get it in then